src/dft/backends/cufft/execute_helper.hpp

```diff
@@ -143,6 +147,27 @@ inline CUstream setup_stream(const std::string &func, sycl::interop_handle ih, c
     return stream;
 }
 
+
+/** Wrap interop API to launch interop host task.
+ * 
+ * @tparam HandlerT The command group handler type
+ * @tparam FnT The body of the enqueued task
+ *
+ * Either uses host task interop API, or enqueue native command extension.
+ * This extension avoids host synchronization after 
+ * the CUDA call is complete.
+ */
+template <typename HandlerT, typename FnT>
+static inline void cufft_enqueue_task(HandlerT&& cgh, FnT&& f) {
```

Comment:
Why is it needed to duplicate this in both cuFFT and rocFFT backends and in various domains (BLAS, LAPACK, FFT)? Can't we have one wrapper used across all domains and backends?